DD 4.15: Dev Envs content (#16180)
docker/docs: desktop/dev-environments/create-dev-env.md

2. Select **Get Started**.
3. Optional: Provide a name for you dev environment.
4. Select **Existing Git repo** as the source and then paste the link of your Git repo subfolder into the field provided.
5. Choose your IDE. You can choose either:
    - **Visual Studio Code**. The Git repository is cloned into a Volume and attaches to your containers. This allows you to develop directly inside of them using Visual Studio Code.
    - **Other**. The Git repository is cloned into your chosen local directory and attaches to your containers as a bind mount. This shares the directory from your computer to the container, and allows you to develop using any local editor or IDE.
6. Select **Continue**.

To launch the application, run the command `make run` in your terminal. This opens an http server on port 8080. Open [http://localhost:8080](http://localhost:8080) in your browser to see the running application.

## Launch from a local folder

1. From **Dev Environments** in Docker Dashboard, select **Create**. The **Create a Dev Environment** dialog displays.
2. Select **Get Started**.

> 3. Optional: Provide a name for you dev environment.

Reviewer: ```suggestion
3. Optional: Provide a name for your dev environment.
```

4. Choose **Local directory** as the source.

> 5. Select **Select** to open the root of the code that you would like to work on.

Reviewer: ```suggestion
5. Select **Select** to open the root directory of the code that you would like to work on.
```

(This read more clearly to me. Somehow "root" without directory made me think about root user or something)

>     A bind mount is created to share the directory from your computer to the container. You can use an editor or IDE of your choice.

Reviewer: ```suggestion
    A directory from your computer is bind mounted to the container, so any changes you make locally will be reflected in the Dev Environment. You can use an editor or IDE of your choice.
```

Clunky phrasing, but I wanted to rephrase "share" which didn't quite seem like the right word here.

>     A directory from your computer is bind mounted to the container, so any changes you make locally is reflected in the dev environment. You can use an editor or IDE of your choice.

Reviewer 2: Suggestion: add an empty line here to create a separate paragraph for the description part of the instruction

```suggestion
   
   A directory from your computer is bind mounted to the container, so any changes you make locally is reflected in the dev environment. You can use an editor or IDE of your choice.
```

>     A directory from your computer is bind mounted to the container, so any changes you make locally will be reflected in the Dev Environment. You can use an editor or IDE of your choice.

Author: ```suggestion
    A directory from your computer is bind mounted to the container, so any changes you make locally is reflected in the dev environment. You can use an editor or IDE of your choice.
```
